test/new-e2e/tests/otel/utils/config_utils.go

```diff
@@ -0,0 +1,72 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package utils contains util functions for OTel e2e tests
+package utils
+
+import (
+	"context"
+	"strings"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+)
+
+// TestOTelAgentInstalled checks that the OTel Agent is installed in the test suite
+func TestOTelAgentInstalled(s OTelTestSuite) {
+	agent := getAgentPod(s)
+	assert.Contains(s.T(), agent.ObjectMeta.String(), "otel-agent")
+}
```

Comment:
should we verify version ?

Reply:
I tried looking in this agent object and didn't see any fields for version. There are some fields for image but that depends on if we're running the tests locally (gcr.io/datadoghq/agent:7-ot-beta) or if it's running in a PR (uses PR image).